iug/src/pages/homepage/index.tsx

```diff
@@ -84,13 +84,11 @@ function Home() {
             hasMore={hasMore}
             loader={
               <div style={{ textAlign: "center" }}>
-                <h4>
-                  <CircularProgress />
-                </h4>
+                <h4>Loading</h4>
               </div>
             }
             scrollableTarget="scrollableDiv"
-            endMessage={<h4></h4>}
```

Comment:
This will cause the lint to fail, see main branch for fixed version unless you've fixed the noProjects variable >_<

Reply:
That is already sorted out in the main brach. I forgot about that, I will cherry pick that tomorrow.